warehouse/templates/manage/account.html

```diff
@@ -600,6 +600,13 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
           {{ event.additional.new_primary }}
         </small>
         {% endif %}
+      {% elif event.tag == "account:email:sent" %}
+      <strong>{% trans %}Email sent{% endtrans %}</strong><br>
+      <small>
+        {% trans %}From:{% endtrans %} {{ event.additional.from_ }}<br>
+        {% trans %}To:{% endtrans %} {{ event.additional.to }}<br>
+        {% trans %}Subject:{% endtrans %} {{event.additional.subject}}
```

Comment:
I think we should probably format these emails like we do in the Account Emails table:

![Screen Shot 2020-06-30 at 5 54 53 PM](https://user-images.githubusercontent.com/294415/86184981-e7193280-bafa-11ea-9343-c6f54fc4e65e.png)

On the fence about whether we should do it for the subject too, but I'm thinking yes.

Reply:
![image](https://user-images.githubusercontent.com/27223486/86292022-97e40800-bbb5-11ea-84d3-1460c04baeda.png)